tests/Doctrine/Tests/ORM/Tools/EntityGeneratorTest.php

```diff
@@ -64,7 +64,7 @@ public function generateBookEntityFixture(array $embeddedClasses = array())
         $metadata->table['uniqueConstraints']['name_uniq'] = array('columns' => array('name'));
         $metadata->table['indexes']['status_idx'] = array('columns' => array('status'));
         $metadata->mapField(array('fieldName' => 'name', 'type' => 'string'));
-        $metadata->mapField(array('fieldName' => 'status', 'type' => 'string', 'default' => 'published'));
+        $metadata->mapField(array('fieldName' => 'status', 'type' => 'string', 'options' => array('default' => 'published')));
```

Comment:
Could you add assertions for these changes? Or was the original assertion wrong?